js/data.js

```diff
@@ -6,3 +6,8 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
```

Comment:
You should add some code to check if there is a value in localStorage when the page loads, and if there is, replace the current value of `data` with that parsed object